src/components/Dashboard/FilingHistoryList/filings/AmalgamationFiling.vue

```diff
@@ -32,7 +32,7 @@
       />
 
       <div
-        v-if="!!tempRegNumber && isStatusCompleted"
+        v-else-if="!!tempRegNumber && isStatusCompleted"
```

Comment:
This makes no functional difference but it's more correct since this code is a set of 3 mutually exclusive blocks.

Note that, if no block is rendered in this template body (see line 23), then the default template body (from FilingTemplate.vue) is used instead.